packages/create/src/generators/app-lit-element-ts/templates/static/index.html

```diff
@@ -7,6 +7,26 @@
   <meta name="Description" content="Put your description here.">
   <base href="/">
 
+  <!--
+    Notice the use of %PUBLIC_URL% in the tags below.
+    It will be replaced with the URL of the `public` folder during the build.
+    Only files inside the `public` folder can be referenced from the HTML.
+
+    Unlike '/favicon.ico' or 'favicon.ico', '%PUBLIC_URL%/favicon.ico' will
+    work correctly both with client-side routing and a non-root public URL.
+    Learn how to configure a non-root public URL by running `npm run build`.
+  -->
+  <link
+    rel="apple-touch-icon"
+    href="%PUBLIC_URL%/logo192.png"
```

Comment:
how will this get replaced by the build?

I think it would be nice to have a solution here that is not tied to a buildstep (e.g. works locally/buildless as well)